traffic_ops/testing/api_contract/v4/request_template.json

```diff
@@ -1,71 +1,75 @@
 {
-    "cdns": [
-        {
-            "name": "test",
-            "domainName": "quest",
-            "dnssecEnabled": false
-        }
-    ],
-    "cachegroup": [
-        {
-            "name": "test",
-            "shortName": "test",
-            "latitude": 38.897663,
-            "longitude": -77.036574,
-            "fallbackToClosest": true,
-            "localizationMethods": [
-                "DEEP_CZ",
-                "CZ",
-                "GEO"
-            ],
-            "typeId": 23
-        }
-    ],
-    "parameters": [
-        {
-            "name": "test",
-            "value": "quest",
-            "configFile": "records.config",
-            "secure": false
-        }
-    ],
-    "roles": [
-        {
-            "name": "test",
-            "description": "quest"
-        }
-    ],
-    "profiles": [
-        {
-            "name": "test",
-            "description": "A test profile for API examples",
-            "cdn": 2,
-            "type": "UNK_PROFILE",
-            "routingDisabled": true
-        }
-    ],
-    "tenants": [
-        {
-            "active": true,
-            "name": "test",
-            "parentId": 1
-        }
-    ],
-    "server_capabilities": [
-        {
-            "name": "RAM"
-        }
-    ],
-    "divisions":[
-        {
-            "name":"test"
-        }
-    ],
-    "regions": [
-		    {
+	"cdns": [
+		{
+			"name": "test",
+			"domainName": "quest",
+			"dnssecEnabled": false
+		}
+	],
+	"cachegroup": [
+		{
+			"name": "test",
+			"shortName": "test",
+			"latitude": 38.897663,
+			"longitude": -77.036574,
+			"fallbackToClosest": true,
+			"localizationMethods": [
+				"DEEP_CZ",
+				"CZ",
+				"GEO"
+			],
+			"typeId": 23
+		}
+	],
+	"parameters": [
+		{
+			"name": "test",
+			"value": "quest",
+			"configFile": "records.config",
+			"secure": false
+		}
+	],
+	"roles": [
+		{
+			"name": "test",
+			"description": "quest"
+		}
+	],
+	"profiles": [
+		{
+			"name": "test",
+			"description": "A test profile for API examples",
+			"cdn": 2,
+			"type": "UNK_PROFILE",
+			"routingDisabled": true
+		}
+
+	],
+	"divisions":[
+		{
+			"name":"test"
+		}
+	],
+	"regions": [
+		{
 			"divisionName": "Quebec",
-			"division": 1,
-			"name": "Montreal"
-    		}
-   	]
+        	"division": 1,
+        	"name": "Montreal"
+		}
+	],
+	"phys_locations": [
+		{
+			"address": "Buckingham Palace",
+			"city": "London",
+			"comments": "Buckingham Palace",
+			"email": "steve.kingstone@royal.gsx.gov.uk",
+			"name": "test",
+			"phone": "0-843-816-6276",
+			"poc": "Her Majesty The Queen Elizabeth Alexandra Mary Windsor II",
+			"regionId": 3,
+			"shortName": "test",
+			"state": "Westminster",
+			"zip": "SW1A 1AA"
+		}
+	]		
```

Comment:
Tests are failing because you deleted the server capabilities and tenants objects from here

Reply:
added them back